docs/vagrant.md

```diff
@@ -11,15 +11,15 @@ Running with Vagrant
 * You can speed up repeated provisioning attempts by mirroring the Splunk package downloads locally:
   1. Download the needed splunk packages locally, in a directory structure mirroring that of download.splunk.com
     * You can find the suffixes of files at the [Splunk download page](http://splunk.com/download)
-    * Copy a download link, it will look like `http://www.splunk.com/page/download_track?file=6.2.5/splunk/linux/splunk-6.2.5-272645-linux-2.6-x86_64.rpm&platform=...`
+    * Copy a download link, it will look like `http://www.splunk.com/page/download_track?file=6.3.3/splunk/linux/splunk-6.3.3-f44afce176d0-linux-2.6-x86_64.rpm&platform=...`
     * The `file` query parameter will be the suffix you add onto `http://download.splunk.com/releases/`
```

Comment:
Minor, go ahead and fix this while you're in here. This doc was missed in PR #106. Should be /products instead of /releases.